packages/types/src/signIn.ts

```diff
@@ -79,7 +82,12 @@ export interface SignInResource extends ClerkResource {
   createMagicLinkFlow: () => CreateMagicLinkFlowReturn<SignInStartMagicLinkFlowParams, SignInResource>;
 }
 
-export type SignInStatus = 'needs_identifier' | 'needs_first_factor' | 'needs_second_factor' | 'complete';
+export type SignInStatus =
+  | 'needs_identifier'
+  | 'needs_first_factor'
+  | 'needs_second_factor'
+  | 'needs_new_password'
```

Comment:
❓ could someone add it also in the FAPI openapi ?
https://github.com/clerkinc/clerk_go/blob/2c67365e804fc6cb651661c034561ae2da5d1b1e/api/client/openapi/schemas/Client.yml#L979